docs/source/reference/io.rst

```diff
@@ -13,8 +13,8 @@ Data Generator
 
    range
 
-Flat File
----------
+Flat File / CSV
```

Comment:
@rxin, this title was from Pandas tho (https://pandas.pydata.org/pandas-docs/stable/reference/io.html#flat-file)